Onboarding/Sources/Onboarding/Onboarding.swift

```diff
@@ -0,0 +1,2 @@
+// The Swift Programming Language
+// https://docs.swift.org/swift-book
```

Comment:
Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579137605)